src/libs/README.md

```diff
@@ -0,0 +1,9 @@
+Jsoncons
+--------
+The library chosen to obtain the value of a metric from the json of cube is ` Jsoncons <https://github.com/danielaparker/jsoncons>`_ . It allows to use JsonPath.
+
+
+
+Prometheus-cpp
+---------------
+The library chosen to create metrics was `Prometheus-cpp <https://github.com/jupp0r/prometheus-cpp.git>`_ .
```

Comment:
If possible, add a brief description of the other libraries contained in this folder.

Reply:
Done :+1: